src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.enums.Color;
+
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].name();
     }
```

Comment:
Creating a new instance of 'Random' each time the method 'getRandomColor' is called can be inefficient. Consider creating a single instance of 'Random' as a class field and reusing it.